docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -409,14 +407,6 @@ def nabisco_cereals():
     return [1, 2, 3]
 ```
 
-#### To multiple groups
```

Comment:
I'm okay with removing this heading, but I would like us to at least keep the error message present in the docs.

We get pinged a lot on what different error messages mean and that there isn't any documentation for them.

Reply:
Got it - I put it back in - let me know what you think.